src/libraries/System.ServiceProcess.ServiceController/README.md

```diff
@@ -0,0 +1,16 @@
+# System.ServiceProcess.ServiceController
+Provides the [`System.ServiceProcess.ServiceController`](https://learn.microsoft.com/en-us/dotnet/api/system.serviceprocess.servicecontroller) class, which allows you to connect to a running or stopped service, manipulate it, or get information about it.
```

Comment:
Minor suggested change:
```suggestion
Provides the [`System.ServiceProcess.ServiceController`](https://learn.microsoft.com/en-us/dotnet/api/system.serviceprocess.servicecontroller) class, which allows you to connect to a Windows service, manipulate it (start, stop, pause, resume), or get information about it.
```

Reply:
I'm keeping this in sync with the NuGet package description. As I don't exactly know what manipulate means, I would rather keep it vague, as is.